experimental/iterators/test/simple-plan-input.mlir

```diff
@@ -1,8 +1,12 @@
-// RUN: mlir-proto-opt %s
+// RUN: mlir-proto-opt %s -convert-iterators-to-std -convert-func-to-llvm | \
```

Comment:
Can we separate this into:
1. a test/Integration that does exactly this
2. keep a small unit test that just calls -convertxxx and FileChecks the produced IR
?

Reply:
Done. Can you briefly check?